### PR TITLE
feat: some faulty styling [don't merge]

### PR DESCRIPTION
**Pull request by thiyagaraj-eightfold**
just for test

### CODE REVIEW

File: src/components/Button/button.module.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
     --button-hover-variant-background-color: var(--primary-color-20);
     --text-color: var(--text-inverse-color);
     background-color: inherit;
-    border: 2px solid rgba(var(--button-primary-default-border-color), 0.01);
+    border: 8px solid rgba(var(--button-primary-default-border-color), 0.01);
```
Comment (dkilgore-eightfold) [inline src/components/Button/button.module.scss:30]: An 8px border is too thick.

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: src/components/Button/button.module.scss (hunk 1 above)
Comment (dkilgore-eightfold) [reply src/components/Button/button.module.scss:30]: This will cause a noticeable flicker on hover and will be jarring.

Comment (thiyagaraj-eightfold) [reply src/components/Button/button.module.scss:30]: This PR is just to test chromatic 😅

Comment (dkilgore-eightfold) [reply src/components/Button/button.module.scss]: Ah, oops.

**Pull request closed without merging**